bolt-sidecar/bin/sidecar.rs

```diff
@@ -55,3 +57,27 @@ async fn main() -> Result<()> {
         }
     }
 }
+
+fn init_tracing() -> eyre::Result<()> {
+    let std_layer = FmtLayer::default().with_writer(std::io::stdout).with_filter(
+        EnvFilter::builder()
+            .with_default_directive("bolt_sidecar=info".parse()?)
+            .from_env_lossy()
+            .add_directive("reqwest=error".parse()?)
+            .add_directive("alloy_transport_http=error".parse()?),
+    );
+    Registry::default().with(std_layer).try_init()?;
+    Ok(())
+}
```

Comment:
Can you explain why you moved this function here?

Reply:
Imo it was weird that tracing was initialised in a function called `init_telemetry_stack`, I didn't expect to find it there. What do you think?

Reply:
I would keep it in telemetry as we use it as a container for "all things monitoring"